src/PhoneInput.php

```diff
@@ -26,7 +26,7 @@ public function init()
         PhoneInputAsset::register($this->view);
         $id = ArrayHelper::getValue($this->options, 'id');
         $jsOptions = $this->jsOptions ? Json::encode($this->jsOptions) : "";
-        $this->view->registerJs("$('#$id').intlTelInput($jsOptions);");
+        $this->view->registerJs("var input = $('#$id'); input.intlTelInput($jsOptions); input.parents('form').on('submit', function() { input.val(input.intlTelInput('getNumber')); });");
```

Comment:
@reastyn I will check it, but I think every `registerJs` adds JS code to a single scope, so `var input` will override all previous phone widget, am I right?